### PR TITLE
Add flag display

### PR DESCRIPTION
**Pull request by Lord-Haji**

### CODE REVIEW

File: chat-plugins/info.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ exports.commands = {
 		}
 
 		let buf = Chat.html`<strong class="username"><small style="display:none">${targetUser.group}</small>${targetUser.name}</strong> `;
+		buf += SG.getFlag(targetUser.userid);
```
Comment (HoeenCoder) [inline chat-plugins/info.js]: Lets just do the flag in profiles and not modify /whois

File: spacialgaze-plugins/SG.js [hunk 2]
```diff
@@ -107,6 +108,14 @@ SG.giveDailyReward = function (userid, user) {
 	Db.DailyBonus.set(userid, [(Db.DailyBonus.get(userid)[0] + 1), Date.now()]);
 };
 
+SG.getFlag = function (userid) {
+	let user = Users(userid);
+	let ip = user.latestIP
+	ip = geoip.lookup(ip);
+	room.add("country: " + ip.name + " / " + ip.code);
```
Comment (HoeenCoder) [inline spacialgaze-plugins/SG.js]: This line was for testing, also `room` isnt defined here

File: spacialgaze-plugins/profile.js [hunk 3]
```diff
@@ -321,10 +309,10 @@ exports.commands = {
 				let profile = '';
 				profile += showBadges(toId(username));
 				profile += '<img src="' + avatar + '" height="80" width="80" align="left">';
-				if (!getFlag(toId(username))) {
+				if (!SG.getFlag(toId(username))) {
```
Comment (HoeenCoder) [inline spacialgaze-plugins/profile.js]: This is looking for a possible `return false` if the new function can't return false (and if its not needed **test this please**) then this if statement is unneeded.

**Review by HoeenCoder — COMMENTED** [review]
Just a few concerns

**Comment by HoeenCoder** [comment]
Also `package.json` is missing the new module

File: spacialgaze-plugins/profile.js (hunk 3 above)
Comment (Lord-Haji) [reply spacialgaze-plugins/profile.js]: **Tested** its not need

**Comment by HoeenCoder** [comment]
Since travis isnt liking this module either, im closing this, if you can fix the issue where travis cannot find the node module file, ill reopen it

**Comment by TheRun98** [comment]
There are still fails with the Travis CI builds

**Comment by Lord-Haji** [comment]
@TheRun98 no it's not

**Comment by Lord-Haji** [comment]
@HoeenCoder Looks like:
scrnsht
Is more resize needed ?

File: spacialgaze-plugins/profile.js [hunk 4]
```diff
@@ -301,31 +299,26 @@ exports.commands = {
...
 			if (Users(userid) && Users(userid).connected) return '<font color = "limegreen"><strong>Currently Online</strong></font>';
 			let seen = Db.seen.get(userid);
 			if (!seen) return '<font color = "red"><strong>Never</strong></font>';
 			return Chat.toDurationString(Date.now() - seen, {precision: true}) + " ago.";
 		}
 
+		function getFlag(userid) {
+			let user = Users(userid);
+			let ip = user.latestIP;
+			ip = geoip.lookup(ip);
```
Comment (HoeenCoder) [inline spacialgaze-plugins/profile.js]: `let ip = geoip.lookup(user.latestIp);`

**Review by HoeenCoder — COMMENTED** [review]
Also too look into: Is it possible for a IP (like `127.0.0.1` maybe) to not have a valid country and err? Make it return `""` if that happens

**Pull request merged**